Dockerfile

```diff
@@ -5,6 +5,9 @@ FROM python:3.9-slim as base
 
 # Upgrade pip to its latest release to speed up dependencies installation
 RUN python -m pip install --upgrade pip
+RUN apt-get update && apt-get install -y python3-opencv
```

Comment:
There is already a block to install system dependencies in the Dockerfile line 14. Add your dependency to the list of installed packages.